src/pages/EditorPage/EditorPanel.jsx

```diff
@@ -506,28 +510,53 @@ const EditorPanel = ({
     <Section wrap id="editor-entity-details-container">
       {!noSelection && (
         <>
-          <EntityDetailsHeader
-            values={nodeIds.map((id) => nodes[id]?.data)}
-            tools={
-              <>
-                <Button
-                  icon="replay"
-                  onClick={handleRevert}
-                  disabled={noSelection}
-                  data-tooltip="Clear changes"
-                />
-                <Button
-                  icon="delete"
-                  onClick={() => onDelete(nodes)}
-                  disabled={noSelection}
-                  data-tooltip="Delete"
-                />
-                <Link to={`/projects/${projectName}/browser`}>
-                  <Button icon="visibility" disabled={noSelection} data-tooltip="View in Browser" />
-                </Link>
-              </>
-            }
-          />
+        <Container>
+          <EntityThumbnailUploader
```

Comment:
The dropzone should span the full height of the right side panel.

![image](https://github.com/user-attachments/assets/534a51e4-cb06-4203-adf1-1db23e2acb44)